pymoto/modules/io.py

```diff
@@ -352,3 +356,69 @@ def _response(self, *args):
             f.write("\n")
 
         self.iter += 1
+
+
+class TransientToVTI(Module):
+    """ Writes transient response vectors to a Paraview VTI file
+
+    This module utilizes a series file to properly associate the correct time with each timestep:
+    https://gitlab.kitware.com/paraview/paraview/blob/v5.5.0/Documentation/release/ParaView-5.5.0.md#json-based-new-meta-file-format-for-series-added
+
+    See also: :attr:`DomainDefinition.write_to_vti()`
+
+    The size of the vectors should be a multiple of ``nel`` or ``nnodes``. Based on their size they are marked as
+    cell-data or point-data in the VTI file. For 2D data (size is equal to ``2*nnodes``), the z-dimension is padded
+    with zeros to have 3-dimensional data. Also, non-transient block-vectors of multiple dimensions
+    (*e.g.* ``(2, 3*nnodes)``) are accepted, which get the suffixed as ``_00``.
+
+    Input Signals:
+      - ``*args`` (`numpy.ndarray`): Vectors to write to VTI. The signal tags are used as name.
+
+    Args:
+        domain: The domain layout
+        transient_tags (list): Tag name of Signal(s) that vary over time
+        saveto (str): Location to save folders with transient responses for specific iterations
+        delta_t (float): Length of timestep
+        interval (int): Iteration interval for saving data
+        scale (float): Scaling factor for the domain
```

Comment:
Typenames not required here, because they are already typehinted in the code. I tend to do put `(optional)` in case of optional arguments (`interval` and `scale`)